lesson15/main.js

```diff
@@ -86,41 +88,49 @@ const displayModal= () => {
   overlay.style.display= "block";
 };
 
+const resetInput = () => {
+  const inputFields = document.getElementsByClassName("js-input");
+
+  for (errorMessage of errorMessages) {
+    errorMessage.textContent =""
+  };
+  for(inputField of inputFields) {
+    inputField.value = ""
+    inputField.style = "border border-slate-600 "
+  };
+}
+
 const closeModal = () => {
-  resetInput()
   modal.style.display="none";
   overlay.style.display="none";
+  resetInput();
 };
 
 const removeModal = () =>  document.getElementById("js-modal-wrapper").remove();
 
-const resetInput = () => {
-  numberInput.value = '';
-  validationMessage.style.display ="none";
-};
-
-const displayValidation = (message) => {
-  validationMessage.textContent = `${message}`;
-  validationMessage.style.display="block";
-}
-
-openListButton.addEventListener("click", () => {
-  const userInput = numberInput.value;
-  if(userInput === ''){
-    displayValidation("Input feild cannot be empty");
-    return;
-  }
-  if (!userInput.match(/^\d+$/)){
-    displayValidation("Input value must be a number");
-    return;
+const checkValidation = (id, serial, message) => {
+  if(id.value === "") {
+    errorMessages[serial].textContent = message;
+    id.style.border = "2px red solid";
+    id.style.margin = "0px";
+  } else {
+    errorMessages[serial].textContent = "";
+    id.style = "border border-slate-600";
```

Comment:
IMO
I found the argument name 'id' a little confusing. I think 'inputArea' etc. would be better but what do you think?

By the way, I thought this validation move was very nice:clap:

Reply:
Thank you for the suggestion!
Changed the parameter's name as suggested.
please see the commit 
6ad544c96b5c4e21ce5ff7a81fa84174f72e7142